pom.xml

```diff
@@ -77,15 +73,19 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
-                <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
-                </configuration>
             </plugin>
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>versions-maven-plugin</artifactId>
```

Comment:
Hva gjør denne?

Reply:
Den gjør det enklere å oppdage utdaterte versjoner i pom.xml. Se gjerne https://www.mojohaus.org/versions-maven-plugin/index.html for mer info. Den som jeg synes er mest nyttig er `mvn versions:display-dependency-updates` som da viser alle utdaterte versjoner.

Reply:
huh, kult. Kan du legge til at det kjøres som et steg i staging-deploy?

Reply:
https://github.com/Informasjonsforvaltning/fdk-metadata-quality-service/pull/34/commits/12694f50b8281f6d9e5b9bff69deb120540509ef

Reply:
Det fungerer jo bra :+1: 

Men er vel null behov for å ha det med i dockerfile? Blir mye mer synlig om det er et eget steg i en workflow, atm så forsvinner det bare bort i alt det andre som skjer under bygging av image.